.drone.yml

```diff
@@ -28,14 +33,15 @@ steps:
         from_secret: aws_secret_access_key
       terraform_tf_states_bucket_name:
         from_secret: terraform_tf_states_bucket_name
-    when:
-      ref:
-        include:
-          - refs/heads/hotfix-*
-          - refs/tags/**
```

Comment:
this means that we are running the CI for all the branches, right? Is the infra able to handle it?

Reply:
No, We configured a condition at pipeline level (instead of step level)
```